tests/kuttl/common/scripts/check_cluster_status.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# List of Northbound pods
+pods=("ovsdbserver-nb-0" "ovsdbserver-nb-1" "ovsdbserver-nb-2")
+ctl_file="ovnnb_db.ctl"
+db_name="OVN_Northbound"
+
+all_good=true  # Initialize status flag
+
+# Check each pod
+for pod in "${pods[@]}"; do
+    # Execute command and capture output
+    output=$(oc exec $pod -n $NAMESPACE -- bash -c "OVS_RUNDIR=/tmp ovs-appctl -t /tmp/$ctl_file cluster/status $db_name")
+
+    # Check if the pod is a cluster member and all pods are mentioned in the connections
+    if ! echo "$output" | grep -q "Status: cluster member" || { for server in "${pods[@]}"; do ! echo "$output" | grep -q "$server" && exit 1; done; }; then
```

Comment:
I'm confused by this conditional, can we please split it somehow and give sub-results some meaningful names, so that it's clear for the reader what is what?

Reply:
I added an example of the output that we're parsing, 
and break the condition to some parts so it'll be readable.
thanks!